### ISSUE TITLE
Expose find_date() parameters through extract_metadata()

### ISSUE THREAD

**Issue by andrewkcarter** [adbar/trafilatura#12]
The parameters for date extraction are hard coded within `extract_metadata()`. In order to get a different date format (ISO-8601 in my case) I need to call it directly, and then I call `extract_metadata()` regardless to capture the other bits. This mean I'm running the date parsing two times, which I would like to optimize out.

Here is what I am doing: 
```
tree = load_html(unicode_body)
publish_date = find_date(tree, extensive_search=True, url=doc_url, outputformat='%Y-%m-%d %H:%M:%S')

meta = extract_metadata(unicode_body)

if meta.author:
    author = meta.author
``` 
It would be nicer to have this (or equivalent):
```
meta = extract_metadata(unicode_body, extensive_search=True, url=doc_url, outputformat='%Y-%m-%d %H:%M:%S')

if meta.author:
    author = meta.author

if meta.date:
    publish_date = meta.date
```

**Comment by adbar** [comment]
Yes, it makes perfect sense, thanks for your input. I just fixed the issue in commit b5ee98075ff3493c859e2db8ce43041b39b8f9d4

**Comment by andrewkcarter** [comment]
Thank you
